Gems/ScriptCanvas/Code/Editor/View/Windows/ScriptEventMenu.cpp

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Editor/Include/ScriptCanvas/Components/EditorGraph.h>
+#include <QFileDialog>
+#include <QMessageBox>
+#include <QObject>
+#include <ScriptCanvas/Bus/RequestBus.h>
+#include <ScriptCanvas/Core/GraphSerialization.h>
+#include <ScriptCanvas/Grammar/ParsingUtilitiesScriptEventExtension.h>
+#include <ScriptEvents/ScriptEventsBus.h>
+
+#include <Editor/View/Windows/ScriptEventMenu.h>
+
+using namespace ScriptCanvas;
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether